Packs/Neosec/Integrations/Neosec/Neosec.yml

```diff
@@ -0,0 +1,126 @@
+category: Network Security
+commonfields:
+  id: Neosec
+  version: -1
+configuration:
+- name: credentials
+  display:
+  required: true
+  defaultvalue:
+  type: 9
+  additionalinfo: The API Key to use for connection
+  displaypassword: API Key
+  hiddenusername: true
+- name: url
+  display: URL
+  required: false
+  defaultvalue: https://neograph.neosec.com
+  type: 0
+  additionalinfo: ""
+- name: tenant_key
+  display: Tenant Key
+  required: false
+  defaultvalue:
+  type: 0
+  additionalinfo: ""
+- name: alert_status
+  display: Fetch alerts with status
+  required: false
+  defaultvalue: Open
+  type: 15
+  additionalinfo: ""
+  options:
+  - Open
+  - Closed
+- name: alert_type
+  display: Fetch alerts with type
+  required: false
+  defaultvalue:
+  type: 16
+  additionalinfo: ""
+  options:
+  - Runtime
+  - Posture
+- name: severities
+  display: Severity of alerts to fetch
+  required: false
+  defaultvalue: Low,Medium,High,Critical
+  type: 16
+  additionalinfo: ""
+  options:
+  - Info
+  - Low
+  - Medium
+  - High
+  - Critical
+- name: max_fetch
+  display: Max incident to fetch
+  required: false
+  defaultvalue: "50"
+  type: 0
+  additionalinfo: ""
+- name: first_fetch
+  display: First fetch time
+  required: false
+  defaultvalue: 3 days
+  type: 0
+  additionalinfo:
+- name: is_tokenized
+  display: De-tokenize alerts
+  required: false
+  defaultvalue: "true"
+  type: 8
+  additionalinfo: ""
+- name: neosec_node_url
+  display: Neosec Node URL
+  required: false
+  defaultvalue:
+  type: 0
+  additionalinfo: ""
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+  defaultvalue: Neosec Incident
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+description: Neosec is reinventing application security. Its pioneering SaaS platform gives security professionals visibility into behavior across their entire API estate. Built for organizations that expose APIs to partners, suppliers, and users, Neosec discovers all your APIs, analyzes their behavior, and stops threats lurking inside.
+display: Neosec
+name: Neosec
+script:
+  commands:
+  - name: neosec-alert-status-set
+    description: Set alert status(Open, Closed)
+    deprecated: false
+    arguments:
+    - name: alert_id
+      description: The alert id(UUID) in the Neosec platform.
+      required: true
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: alert_status
+      description: The alert status, the options are "Open" or "Closed".
+      required: true
+      isArray: false
+      defaultValue: ""
+      predefined:
+      - Open
+      - Closed
```

Comment:
I see you had the intention to have this as a predefined argument, I think adding this field will fix the issue and show the options in XSOAR
`auto: PREDEFINED`